controllers/base.go

```diff
@@ -23,23 +23,23 @@ type BaseController struct {
 	actionName string
 }
 
-func (base *BaseController) Prepare() {
-	controller := &base.Controller
+func (b *BaseController) Prepare() {
```

Comment:
updated this to align with our convention to use abbreviation as the receiver name